ui/components/institutional/confirm-remove-jwt-modal/confirm-remove-jwt-modal.tsx

```diff
@@ -30,10 +30,15 @@ type AuthDetails = {
   refreshToken?: string;
 };
 
+type LabelItem = {
+  key: string;
+  value: string;
+};
+
 type CustodyAccountDetail = {
   address: string;
   name: string;
-  labels: string[];
```

Comment:
This is not equivalent. Has the log changed for generating these `labels`?

Reply:
In a previous PR, I introduced `ConfirmRemoveJwtModal` without typing `AccountList` and did not thoroughly review all the variables and dependencies. This PR addresses that by properly typing AccountList and fixing all its dependencies.